File: dotnet/ArqLibrarianClassic/ArqLibrarianClassic/Library/SingleRating.cs
using System;

namespace ArqLibrarianClassic.Library
{
    public class SingleRating
    {
        public int Rating { get; }
        private DateTime when;

        public SingleRating(int rating)
        {
            this.Rating = rating;
            this.when = DateTime.Now;
        }
    }
}

File: java/ArqLibraryClassic/ArqLibraryLayered/src/bnsit/ms/arq/library/SingleRating.java
package bnsit.ms.arq.library;

import java.util.Date;

public class SingleRating {

    private int rating;

    private Date when;

    public SingleRating(int rating)
    {
        this.rating = rating;
        this.when = new Date();
    }

    public int getRating() {
        return rating;
    }
}
